### DISCUSSION TITLE
Welcome to RunestoneComponents Discussions!

### DISCUSSION THREAD

**Discussion by bnmnetp** [RunestoneInteractive/RunestoneComponents#1110] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bjones1** [comment]
Hello everyone, my name is Bryan Jones. I teach a course on microprocessors in the Department of Electrical and Computer Engineering at Mississippi State University. I've written a Runestone-based book to accompany the course and contributed to Runestone as well. I look forward to meeting everyone else!
